x-pack/plugins/ml/public/application/util/chart_utils.d.ts

```diff
@@ -5,4 +5,13 @@
  * 2.0.
  */
 
+import { CHART_TYPE } from '../explorer/explorer_constants';
+
 export declare function numTicksForDateFormat(axisWidth: number, dateFormat: string): number;
+export declare function getChartType(config: any): typeof CHART_TYPE[keyof typeof CHART_TYPE];
```

Comment:
Could use `ChartType` suggested for `explorer_constants.ts`.

Reply:
Updated here [`4c0419c` (#94396)](https://github.com/elastic/kibana/pull/94396/commits/4c0419cb0a7f21e41e0308d0a3e6d3566766022d)